.config/forest.dic

```diff
@@ -9,3 +9,8 @@ Merkle
 Filecoin
 APIs
 CBOR
+codec
+blake2b
+serde_bytes
+>
+<
```

Comment:
`serde_bytes`, `<` and `>` aren't words and shouldn't be in the dictionary. You can rewrite the comments such that any code is marked as being code.